webservices/partition/sched_a.py

```diff
@@ -40,6 +42,7 @@ def column_factory(cls, parent):
                 parent.c.contbr_id,
                 parent.c.cmte_id,
             ).label('clean_contbr_id'),
+            sa.cast(sa.func.get_transaction_year(parent.c[cls.transaction_date_column], parent.c.rpt_yr), sa.Numeric(4, 0)).label('transaction_two_year_period'),
```

Comment:
Sorry, the error is `child table "ofec_sched_a_1977_1978_tmp" has different type for column "transaction_two_year_period"` without the casting.  I thought perhaps the return type of the function was causing the problem but even changing it to `numeric` doesn't seem to matter.